packages/fiori-freestyle-writer/src/defaults.ts

```diff
@@ -28,4 +28,6 @@ export function setDefaults(ffApp: FreestyleApp<unknown>): void {
     if (ffApp.template.type === TemplateType.Basic) {
         setBasicTemplateDefaults(ffApp.template.settings as BasicAppSettings);
     }
+    // All fiori-freestyle apps should use load reuse libs, unless explicitly overridden
+    ffApp.appOptions = Object.assign({ loadReuseLibs: true }, ffApp.appOptions);
```

Comment:
The `loadReuseLibs`  param only applies to the index.html and not to the writing of `flpsandbox.html` and `locate-reuse-libs.js`.
Should be be considered in those cases too or just removed as a param?

Reply:
The `locate-reuse-libs.js` file is taken care of by the `loadRuseLibs` param but you are correct that the `flpSandbox.html` needs a condition around adding the script reference, thx. Well spotted.